.github/workflows/coverage.yml

```diff
@@ -38,25 +38,20 @@ jobs:
         sudo apt-get update
         sudo apt-get -y install openmpi-bin libopenmpi-dev libopenblas-dev
 
-    # TODO(Crissman): Remove click after issue with allennlp import resolved
-    # https://github.com/optuna/optuna/pull/2665
-    - name: Install click 7.1.2 until allennlp import issue resolved
-      run: pip install --use-deprecated=legacy-resolver --progress-bar off click==7.1.2
-
     - name: Install
       run: |
         python -m pip install --upgrade pip
-        pip install --use-deprecated=legacy-resolver --progress-bar off -U setuptools
+        pip install --progress-bar off -U setuptools
 
         # Install minimal dependencies and confirm that `import optuna` is successful.
-        pip install --use-deprecated=legacy-resolver --progress-bar off .
+        pip install --progress-bar off .
         python -c 'import optuna'
         optuna --version
 
-        pip install --use-deprecated=legacy-resolver --progress-bar off .[tests]
-        pip install --use-deprecated=legacy-resolver --progress-bar off .[optional]
-        pip install --use-deprecated=legacy-resolver --progress-bar off .[integration] -f https://download.pytorch.org/whl/torch_stable.html
-        pip install --use-deprecated=legacy-resolver --progress-bar off .[codecov]
+        pip install --progress-bar off .[tests]
```

Comment:
Can we merge these installations? In sphinx-build, a single `pip install` seems to install multiple extras.
(The same applies to other parts in this PR)

https://github.com/optuna/optuna/pull/2779/files#diff-e4697a2845e32e857d02b8540419a37459fa603b4db72167817367038b4f5a64R87

Reply:
Thank you for your feedback! I just removed the option. Could you elaborate a little bit more about your comment?

Reply:
Sorry for the vague comment. I was just wondering if we could install external dependencies at the same time.

```
        pip install --progress-bar off .[tests]
        pip install --progress-bar off .[optional]
        pip install --progress-bar off .[integration] -f https://download.pytorch.org/whl/torch_stable.html
        pip install --progress-bar off .[codecov]
```

to

```
        pip install --progress-bar off .[tests,optional,integration,codecov] -f https://download.pytorch.org/whl/torch_stable.html
```

I suggested it because it looks trivial and could be done in this PR, but it may beyond the scope of this PR.

Reply:
Sure. I think we can do it. But this kinda separation is supposed to be useful when we investigate some CI errors. Anyway, either is fine for me.

Reply:
I see. It's reasonable for me. Thank you for the answer.